lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -1826,6 +1826,199 @@ class ConvertAtenBatchNormOp : public OpConversionPattern<AtenBatchNormOp> {
 };
 } // namespace
 
+namespace {
+class ConvertAtenInstanceNormOp
```

Comment:
nit: I think this pattern can be written as a decomposition (the linalg.generics seem to recreate the behavior of torch ops we already support). Using a decomposition would simplify the pattern and have the added benefit that the other 2 backends in torch-mlir get the lowering for free.

Reply:
how do I write a decomposition?

Reply:
Here is an example that does a norm operation: https://github.com/llvm/torch-mlir/pull/1332